lib/ansible/modules/packaging/os/zypper_repository.py

```diff
@@ -52,6 +53,12 @@
         type: bool
         default: 'yes'
         aliases: [ "refresh" ]
+    extra_args_precommand:
+       version_added: "2.8"
+       required: false
+       description:
+         - Add additional global target options to C(zypper).
```

Comment:
Since this messes with the internals of the module, maybe add a warning/comment like
```suggestion
         - Add additional global target options to C(zypper).
         - This is most useful for options modifying paths like C(--root), C(--installroot) or the C(--*-dir) flags. For features like gpg auto import please use the existing option.
```

Reply:
Thanks. Done